src/main/java/org/teamresistance/frc/autonomous/AutoPingController.java

```diff
@@ -0,0 +1,32 @@
+package org.teamresistance.frc.autonomous;
+
+import org.teamresistance.frc.subsystem.Controller;
+import edu.wpi.first.wpilibj.PIDSourceType;
+import org.strongback.control.SoftwarePIDController;
+import org.teamresistance.frc.Pose;
+import org.teamresistance.frc.subsystem.Controller;
+import org.teamresistance.frc.util.SynchronousPID;
+
+
+/**
+ * Created by owner on 1/25/2017.
+ */
+public class AutoPingController implements Controller {
+  private SynchronousPID pingPID;
+  private static final double TOLERANCE = 2;
+  private static final double KP = 0.0;
+  private static final double KD = 0.0;
+  private static final double KI = 0.0;
+  private static final double FF = 0.0;
+  double targetDistance;
```

Comment:
An instance of this class will only ever have one `targetDistance` with the aforementioned constructor changes, and so this variable can be made `final` to enforce that fact. You can also make it `private` since now we'll only need to access it internally.